akka-typed-tests/src/test/java/akka/typed/javadsl/BehaviorBuilderTest.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.typed.javadsl;
+
+import akka.typed.Behavior;
+import akka.typed.Terminated;
+import org.junit.Test;
+import org.scalatest.junit.JUnitSuite;
+
+import static akka.typed.javadsl.Actor.same;
+import static akka.typed.javadsl.Actor.stopped;
+
+public class BehaviorBuilderTest extends JUnitSuite {
+    interface Message {
+    }
+
+    static class One implements Message {
+        public void foo() {}
+    }
+
+    @Test
+    public void shouldCompile() {
+        Behavior<Message> b = BehaviorBuilder.<Message>create()
```

Comment:
or put the behaviorBuilder factory in Actor, similar i AbstractActor. Reduce number of imports.